src/style.css

```diff
@@ -0,0 +1,133 @@
+.page {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-weight: 300;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  width: 198px;
+  height: 406px;
+}
+
+.card__img {
+  width: 160px;
+  height: 134px;
+  margin: 32px 19px 40px;
+  display: flex;
+  justify-content: center;
+}
+
+.card__content {
+  margin: 0 16px 5px;
+  display: flex;
+  flex-wrap: 1;
+  font-weight: 500;
+  font-size: 12px;
+  color: #060b35;
+  line-height: 18px;
+}
+
+.card__prescription {
+  margin: 0 16px 18px;
+  font-weight: 400;
+  font-size: 10px;
+  color: #616070;
+}
+
+.rating {
+  margin: 0 16px 23px;
+  display: flex;
+  justify-content: space-between;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars-item {
+  background-image: url(./images/star.svg);
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+}
+
+.stars--4 > .stars-item:nth-child(-n + 4) {
+  background-image: url(./images/star-active.svg);
+}
+
+.rating__comments {
+  display: flex;
+  font-weight: 400;
+  font-size: 10px;
+  color: #060b35;
+  align-items: center;
+  text-align: center;
+}
+
+.price-block {
+  margin: 0 16px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.price-block__prescription {
+  font-weight: 400;
+  font-size: 12px;
+  color: #616070;
+}
+
+.price-block__price {
+  font-weight: 700;
+  font-size: 16px;
+  color: #060b35;
+}
+
+.button-block {
+  margin: 16px;
+  display: flex;
+  justify-content: center;
+  text-align: center;
+  align-items: center;
+}
+
+.button-block__button {
+  padding: 0;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background-color: #00acdc;
+  border-radius: 5px;
+  border: none;
+  width: 166px;
+  height: 40px;
+}
+
+.button-block__button:hover {
+  background: #fff;
+  border: 1px solid #00acdc;
```

Comment:
you can transparent border and change just border color on hover
9 point from [checklist](https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md)